README.md

```diff
@@ -16,22 +16,22 @@ Read [Apache ORC](https://orc.apache.org/) in Rust.
 
 ## Current Support
 
-| Column Encoding           | Read | Write | Rust Type             | Arrow  DataType         |
-| ------------------------- | ---- | ----- | --------------------- | ----------------------- |
-| SmallInt, Int, BigInt     | ✓    |       | i16, i32, i64         | Int16, Int32, Int64     |
-| Float, Double             | ✓    |       | f32, f64              | Float32, Float64        |
-| String, Char, and VarChar | ✓    |       | string                | Utf8                    |
-| Boolean                   | ✓    |       | bool                  | Boolean                 |
-| TinyInt                   | ✓    |       | i8                    | Int8                    |
-| Binary                    | ✓    |       | Vec\<u8\>             | Binary                  |
-| Decimal                   | ✗    |       |                       |                         |
-| Date                      | ✓    |       | chrono::NaiveDate     | Date32                  |
-| Timestamp                 | ✓    |       | chrono::NaiveDateTime | Timestamp(Nanosecond,_) |
-| Timestamp instant         | ✗    |       |                       |                         |
-| Struct                    | ✓    |       |                       | Struct                  |
-| List                      | ✓    |       |                       | List                    |
-| Map                       | ✓    |       |                       | Map                     |
-| Union                     | ✗    |       |                       |                         |
+| Column Encoding           | Read | Write | Arrow DataType          |
+| ------------------------- | ---- | ----- | ----------------------- |
+| SmallInt, Int, BigInt     | ✓    |       | Int16, Int32, Int64     |
+| Float, Double             | ✓    |       | Float32, Float64        |
+| String, Char, and VarChar | ✓    |       | Utf8                    |
+| Boolean                   | ✓    |       | Boolean                 |
+| TinyInt                   | ✓    |       | Int8                    |
+| Binary                    | ✓    |       | Binary                  |
+| Decimal                   | ✗    |       |                         |
+| Date                      | ✓    |       | Date32                  |
+| Timestamp                 | ✓    |       | Timestamp(Nanosecond,_) |
+| Timestamp instant         | ✗    |       |                         |
+| Struct                    | ✓    |       | Struct                  |
+| List                      | ✓    |       | List                    |
+| Map                       | ✓    |       | Map                     |
+| Union                     | ✗    |       |                         |
```

Comment:
I removed the Rust Type column as we'll likely be focused on interop with Arrow.

A more general API for non-arrow Rust use (like parquet ColumnReader API) can be implemented later if their is use case for it, for now we'll focus on Arrow if that's alright